source/sfp/1_water/sfp_115_nl_abiotiek_multimeter/NEWS.md

```diff
@@ -0,0 +1,22 @@
+# Wijzigingen t.o.v. vorige versies
+
+## [2024.04](../2024.04/index.html)
+
+-   Dit is de eerste versie van het protocol
```

Comment:
Hier kan je vermelden dat het een omzetting is van bestaand Word protocol naar nieuwe systeem en verwijzen naar de oude versienummer en publicatiedatum van het omgezette protocol.

Reply:
Is gebeurd!